src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2766,6 +2830,86 @@ export async function getGenomicDataAsClinicalData(
     });
 }
 
+export function convertGenericAssayDataBinsToClinicalDataBins(
```

Comment:
why do we have to convert to "clinical data" bins?   would it not be better to generalize bins?

Reply:
@kalletlak I think there may have some discussions when you implemented the `gene specific charts`. I am wondering whether you know the reason why we decided to convert other data bins to `clinical data bins`?

Reply:
@kalletlak if perhaps the first implementation of this was for clinical data and now we are using that for other forms of data, would be a lot of work to make the implementation generic?

Reply:
Initially it was just for clinical data. And then I added for gene specific charts. Since it is evolving it make sense to implement it in a generic way.

Reply:
Got it, thanks @kalletlak, will update this to a generic solution.

Reply:
Updated to use a generic type `DataBin`